lib/steep/cli.rb

```diff
@@ -113,6 +113,14 @@ def process_stats
           opts.banner = "Usage: steep stats [options] [sources]"
 
           opts.on("--steepfile=PATH") {|path| check.steepfile = Pathname(path) }
+          opts.on("--format=FORMAT", "Specify output format: csv, table") do |format|
+            case format
+            when "csv", "table"
+              check.format = format
+            else
+              stderr.puts "--format option is ignored because it has unknown value: `#{format}`"
```

Comment:
[IMO] Since `ArgumentError` occurs on an invalid `--log-level` value, it seems kind to me to raise `ArgumentError` also on an invalid `--format` value.

Error:

```console
$ bundle exec steep stats --log-level=fata
#<ArgumentError: invalid log level: data>
```

No error:

```console
$ bundle exec steep stats --log-level=fatal --format=tabl
--format option is ignored because it has unknown value: `tabl`
# Calculating stats:

 Target  File                                             Status   Typed calls  Untyped calls  All calls  Typed %
------------------------------------------------------------------------------------------------------------------
...
```

Reply:
Good point! Will do. 💪